sound/soc/sof/intel/hda.h

```diff
@@ -367,6 +367,9 @@ struct sof_intel_hda_dev {
 	/* if position update IPC needed */
 	u32 no_ipc_position;
 
+	/* the total playback + capture stream number */
```

Comment:
clarify this is the maximum total playback + capture streams supported by the hardware.
This is NOT the current total number of streams in use.

Reply:
correct, maybe something like "num_of_streams_supported"